<?xml version="1.0"?>
<launch>
    <param name="use_sim_time" value="true"/>
    <group ns="costmap_node">
        <rosparam file="$(find lunabot_nav)/config/local_costmap.yml" command="load" ns="local_costmap" />
    </group>

    <node name="costmap_node" pkg="lunabot_nav" type="costmap_node" output="screen"/>

    <!--
    <node name="global_planner" pkg="lunabot_nav" type="global_planner" output="screen"/>
    -->
</launch>